### ISSUE TITLE
[2.2.2]Monitoring - project owner/member should be able to see Grafana when project monitoring is on

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#19404]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- log in as admin to enable the cluster level monitoring
- create a new project p1 and **enable the project monitoring**
- enable Github Auth and add a new user user1 as the project owner/member
- log in as user1, go to view any workload in the project 

**Result:**
- user1, as the project owner/member, cannot see the Grafana icon on workload, pod, and container
<img width="1262" alt="screenshot" src="https://user-images.githubusercontent.com/6218999/55591916-bece3880-56eb-11e9-8548-e1c73a130945.png">

- trying to access to Grafana page will end up with this error
```
services \"http:access-grafana:80\" is forbidden: User \"u-rudxpz7guh\" cannot get resource \"services/proxy\" in API group \"\" in the namespace \"cattle-prometheus\"
```
<img width="1463" alt="screenshot" src="https://user-images.githubusercontent.com/6218999/55591963-e0c7bb00-56eb-11e9-914d-6b5c859a872a.png">

**Expected Result**
the project owner/member should be able to see Grafana icon on workload, pod, and container when cluster monitoring is on.
<img width="891" alt="screenshot" src="https://user-images.githubusercontent.com/6218999/55592332-f5f11980-56ec-11e9-88e0-11755627b463.png">

**Other details that may be helpful:**
This is a regression in v2.2.2-rc2. 
The feaure works in v2.2.1

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.2.2-rc2
- Installation option (single install/HA): 
single install

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Infrastructure Provider, EC2
- Machine type (cloud/VM/metal) and specifications (CPU/memory): 2 vCPU, 8GB memory
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.7", GitCommit:"6f482974b76db3f1e0f5d24605a9d1d38fad9a2b", GitTreeState:"clean", BuildDate:"2019-03-25T02:52:13Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.5", GitCommit:"2166946f41b36dea2c4626f90a77706f426cdea2", GitTreeState:"clean", BuildDate:"2019-03-25T15:19:22Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
18.9.4
```

**Comment by loganhz** [comment]
Please enable project level monitoring. Otherwise, you can not see it. It is by design.

**Comment by jiaqiluo** [comment]
@loganhz @sangeethah I edited the issue to mention that the issue does exist when the project monitoring is on.  I realized that I missed this info before.

**Comment by alena1108** [comment]
Can be tested with `rancher/rancher:v2.2.0-rc4`

**Comment by jiaqiluo** [comment]
The bug fix is validated on v2.2.2-rc5 
The same steps as the issue described

Result:
the project owner/member can see Grafana icon on workload, pod, and container when cluster monitoring is on
